executor/read/collecting_stage_executor.rs

```diff
@@ -215,7 +216,10 @@ impl CollectorTrait for SortCollector {
     fn into_iterator(self, context: &ExecutionContext<impl ReadableSnapshot>) -> CollectedStageIterator {
         let Self { sort_on, collector } = self;
         let unsorted = collector.unwrap();
-        let sorted_indices = unsorted.indices_sorted_by(context, &sort_on).into_iter().peekable();
+        let profile = context.profile.profile_stage(|| String::from("Sort"), 0); // TODO executable id
```

Comment:
There's a question of what to use as an executable id (or if we even need one) for stages not associated with an executable. There are a few other places this comes up, notably in Reduce and schema queries (which don't have stages).

Reply:
I guess we don't need one, just use 0 for now i spose